egs/commonvoice/ASR/RESULTS.md

```diff
@@ -1,4 +1,73 @@
 ## Results
+
+### Commonvoice Cantonese (zh-HK) Char training results (Zipformer)
+
+See #1542 for more details.
+
+Number of model parameters:  72526519, i.e., 72.53 M
+
+The best CER, for CommonVoice 16.1 (cv-corpus-16.1-2023-12-06/zh-HK) is below:
+
+|                      |  Dev  | Test |        Note        |
+|----------------------|-------|------|--------------------|
+|    greedy_search     | 1.17  | 1.22 | --epoch 24 --avg 5 |
+| modified_beam_search | 0.98  | 1.11 | --epoch 24 --avg 5 |
+|   fast_beam_search   | 1.08  | 1.27 | --epoch 24 --avg 5 |
+
+When doing the cross-corpus validation on MDCC (w/o blank penalty),
```

Comment:
Could you put a link to `MDCC`. Otherwise, it is not clear what `MDCC` stands for.